app/models/page.rb

```diff
@@ -1,2 +1,61 @@
 class Page < ActiveRecord::Base
+  def shopper_report_v2
+    # not align equles
+
+    asdf = Time.zone.now
+    asdf23r = Time.zone.now
+    asdasdgff = Time.zone.now
+    asdasdf = Time.zone.now
+
+
+    # line breaks
+    type1 = [
+      1,
+      2,
+      3
+    ]
+
+    type2 = [
+        1,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.